### PR TITLE
add utility functions to check and call master contract

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**

### CODE REVIEW

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                peers.forEach {
+                    println("$it is peer: ${master.isPeer(it).send()}");
+                }
+                logger.info { "xor ${master.xorTokenInstance().send()}" }
+            }
+            .failure { ex ->
+                logger.error(ex.toString())
+                System.exit(1)
+            }
+}
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: :-)

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                DeployHelper(
+                        ethereumConfig,
+                        passwordConfig
+                )
+            }
+            .map { deployHelper ->
+                logger.info { "Load master contract at $masterContractAddress" }
+                val master = deployHelper.loadMasterContract(masterContractAddress)
+                peers.forEach {
+                    println("$it is peer: ${master.isPeer(it).send()}");
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: Do we need this?

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        .map { deployHelper ->
+            logger.info { "Load master contract at $masterContractAddress" }
+            val master = deployHelper.loadMasterContract(masterContractAddress)
+            peers.forEach {
+                println("$it is peer: ${master.isPeer(it).send()}");
+            }
+            logger.info { "xor ${master.xorTokenInstance().send()}" }
+        }
+        .failure { ex ->
+            logger.error(ex.toString())
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: `toString` returns no useful information in the `Exception` class. I'd rather use `logger.error("You message",e)`. If you do like that, you log an exception stack trace as well.

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            DeployHelper(
+                ethereumConfig,
+                passwordConfig
+            )
+        }
+        .map { deployHelper ->
+            logger.info { "Load master contract at $masterContractAddress" }
+            val master = deployHelper.loadMasterContract(masterContractAddress)
+            peers.forEach {
+                println("$it is peer: ${master.isPeer(it).send()}");
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: Do we need this?

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterWithdrawToWallet.kt [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+                beneficiary,
+                Numeric.hexStringToByteArray(txHash),
+                vv,
+                rr,
+                ss,
+                beneficiary
+            ).send()
+        }
+        .failure { ex ->
+            logger.error(ex.toString())
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterWithdrawToWallet.kt]: The same thing.

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: yes, to check all notary addresses in contract

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt (hunk 3 above)
Comment (Alexey-N-Chernyshov) [reply eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: yes, to check all notary addresses in contract

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt (hunk 4 above)
Comment (Alexey-N-Chernyshov) [reply eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: yes, to check all notary addresses are in contract

**Review by Mingela — APPROVED** [review]

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt (hunk 4 above)
Comment (dolgopolovwork) [reply eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: No, I mean `println`. We can just log it.

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            }
+            logger.info { "xor ${master.xorTokenInstance().send()}" }
+        }
+        .failure { ex ->
+            logger.error("Master contract call exception", ex.toString())
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterIsPeer.kt]: No, it is not going to work. `toString`, as I told you before, returns no useful information at all. Please, use the following form `logger.error("Message",ex)`

File: eth/src/main/kotlin/com/d3/eth/util/CallMasterWithdrawToWallet.kt [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+                beneficiary,
+                Numeric.hexStringToByteArray(txHash),
+                vv,
+                rr,
+                ss,
+                beneficiary
+            ).send()
+        }
+        .failure { ex ->
+            logger.error("Master contract call exception", ex.toString())
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/util/CallMasterWithdrawToWallet.kt]: The same

**Review by dolgopolovwork — APPROVED** [review]

**Pull request merged**